### PR TITLE
Improve wording and fix a typo

### PR DESCRIPTION
**Pull request by Polaris000**

### CODE REVIEW

File: metrics/issues-open.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Mandatory:
 ### Bugzilla Case
 
 * Platform: The platform provides several statuses and those depends on the configuration of each project. By default Bugzilla has a [workflow](https://www.bugzilla.org/docs/3.6/en/html/lifecycle.html) where the open statuses may be considered: all but *closed* and *resolved*.
-* Case Definition: Open issues need the statuses parameter to be defined. By default this may be all statuses but *closed* and *resolved*. 
+* Case Definition: Open issues need the statuses parameter to be defined. By default, this may be all statuses, except for  *closed* and *resolved*. 
 
 ## 2. Use Cases
 
-* Many people would like an at-a-glance visualization of all the open issues in a project, this visualization would provide that without having to look inside every sub-project in the repository and sum the issues manually
+* Many people would like an at-a-glance visualization of all the open issues in a project. This visualization would provide that without having to look inside every sub-project in the repository and sum the issues manually
```
Comment (Polaris000) [inline metrics/issues-open.md]: I am not too sure about what is meant by this particular use case, and hence, not confident about this change. Please suggest a better change if the current one deviates from the meaning of the paragraph.

File: metrics/issues-open.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Note: do not confuse this with the opened tickets in a specific period of time r
 
 Mandatory: 
 * Point in time. This is the time for which the snapshot of the open issues is computed.
-* Statuses. These are the statuses considered as open. Platform and project dependant.
+* Statuses. These are the statuses considered as open. This component is platform and project dependent.
```
Comment (jgbarah) [inline metrics/issues-open.md]: Thanks for noticing the typo with "dependant". For the rest of this change, I think I prefer the terser form. So, please change it to "Platform and project dependent".

File: metrics/issues-open.md (hunk 1 above)
Comment (jgbarah) [reply metrics/issues-open.md]: This change is ok, thanks.

File: metrics/issues-open.md (hunk 2 above)
Comment (Polaris000) [reply metrics/issues-open.md]: I have made the required changes. Please have a look.

File: metrics/issues-open.md (hunk 1 above)
Comment (foundjem) [reply metrics/issues-open.md]: Seems good to me.

**Pull request merged**